packages/core/theme/src/components/select.ts

```diff
@@ -9,7 +9,7 @@ const select = tv({
     label: "block text-small font-medium text-foreground-500 pointer-events-none",
     mainWrapper: "w-full flex flex-col",
     trigger:
-      "relative px-3 gap-3 w-full inline-flex flex-row items-center shadow-sm outline-none tap-highlight-transparent",
+      "relative px-3 gap-3 w-full inline-flex flex-row items-center shadow-sm outline-none tap-highlight-transparent pr-10",
     innerWrapper:
       "inline-flex h-full w-[calc(100%_-_theme(spacing.unit-6))] items-center gap-1.5 box-border",
```

Comment:
@jguddas this line already handles the overlaps `innerWrapper: "... w-[calc(100%_-_theme(spacing.unit-6))]"`, we could use either the padding-right or the width

Reply:
I will build an example when I get time, I definitely have overlap when I don't add it.

Reply:
Ok maybe we could remove the `width` then, and only use the padding

Reply:
Ideally we would use inline rather than absolute, this would be a bit bigger change tho.

Reply:
What do you mean by inline?

Reply:
Remove display: absolute

Reply:
Oh got it you mean in the `selectorIcon`, It will be more complex because the `innerWrapper` slot can grow with the content you we need to keep the selector icon always in the correct position, Let's go first with padding and removing the `w-[calc(100%_-_theme(spacing.unit-6))]`